Tests/SentryTests/Networking/SentryHttpTransportTests.swift

```diff
@@ -233,8 +233,13 @@ class SentryHttpTransportTests: XCTestCase {
 
         XCTAssertEqual(1, fixture.requestManager.requests.count)
 
-        let actualRequest = fixture.requestManager.requests.last
-        XCTAssertEqual(fixture.userFeedbackRequest.httpBody, actualRequest?.httpBody, "Request for user feedback is faulty.")
+        let actualData = fixture.requestManager.requests.last?.httpBody
+        let expectedData = fixture.userFeedbackRequest.httpBody
+        let decompressedActualData = sentry_unzippedData(actualData!)
+        let decompressedExpectedData = sentry_unzippedData(expectedData!)
+        let actualEnvelope = SentrySerialization.envelope(with: decompressedActualData!)
+        let expectedEnvelope = SentrySerialization.envelope(with: decompressedExpectedData!)
+        try! EnvelopeUtils.assertEnvelope(expected: expectedEnvelope!, actual: actualEnvelope!)
```

Comment:
Prefer using `try` and marking the test function as `throws` so it doesn't take down the test run if it fails.